packages/common/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [2.1.2] - 2023-05-26
```

Comment:
Can you use `chan` please. It should create a link down the bottom to the diff for `[2.1.2]`